airflow/assets/__init__.py

```diff
@@ -266,20 +269,43 @@ class Asset(os.PathLike, BaseAsset):
     uri: str
     group: str
     extra: dict[str, Any]
+    watchers: list[BaseTrigger] = []
```

Comment:
do we need to use field instead? not sure how attrs deals with default values

Reply:
It does not handle default values well. This should use `attrs.field(factory=list)` instead.

Reply:
Although I do not think we need a default at all? The `__init__` implementation always provides a value anyway.

Reply:
Makes sense, removed